recipes/default.rb

```diff
@@ -48,15 +43,15 @@
                     iptables -t nat -A OUTPUT -d localhost -p tcp -m tcp --dport 53 -j REDIRECT --to-ports 8600
                     iptables-save | tee /etc/iptables/rules.v4
                     ip6tables-save | sudo tee /etc/iptables/rules.v6
-                    sed -i "s/#DNS=/DNS=127.0.0.2/g" /etc/systemd/resolved.conf
+                    sed -i "s/#DNS=/DNS=#{my_private_ip()}/g" /etc/systemd/resolved.conf
                     sed -i "s/#Domains=/Domains=~#{node['consul']['domain']}/g" /etc/systemd/resolved.conf
                 EOH
             end
 
             if node['install']['localhost'].casecmp?("true")
-                dnsmasq_ip = "127.0.0.2"
+                dnsmasq_ips = "127.0.0.2,127.0.0.1"
```

Comment:
Why do we need both 127.0.0.2 and 127.0.0.1 ? I guess we can squash them. I think `my_private_ip()` returns 127.0.0.1 when localhost==true

Reply:
Squashed.